lexicons/community/atprotocol/map.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "community.atprotocol.geomarker.map",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["locations"],
+        "properties": {
+          "label": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 800,
+            "maxGraphemes": 400
+          },
+          "description": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 600,
+            "maxGraphemes": 1200
+          },
+          "mainEntry": {
```

Comment:
What is a main entry here? Are we imagining a pointer to a marker that becomes the "cover" of the map? Can it be something that's not a pointer to a marker?